embulk-output-postgresql/src/test/java/org/embulk/output/postgresql/PostgreSQLTests.java

```diff
@@ -38,10 +37,15 @@ public static void execute(String sql)
         int code;
         try {
             Process process = pb.start();
-            ByteStreams.copy(process.getInputStream(), System.out);
+            InputStream inputStream = process.getInputStream();
+            byte[] buffer = new byte[8192];
+            int readSize;
+            while ((readSize = inputStream.read(buffer)) != -1) {
+                System.out.write(buffer,0,readSize);
```

Comment:
The coding style convention prefers to have a whitespace (`' '`) after comma (`','`).